File: TFTP/Server/src/main/java/impl/tftp/CommandBcast.java
package bgu.spl171.net.impl.tftp;

public class CommandBcast extends Command {

	private String fileName;
	private byte type;
	
	public CommandBcast(byte type, String fileName) {
		super((short) 9);
		this.type = type;
		this.fileName = fileName;
	}
	
	public String getName(){
		return fileName;
	}
	
	public byte getType(){
		return type;
	}

	
}

File: TFTP/Client/src/CommandBcast.cpp
#include "CommandBcast.h"

using namespace std;

CommandBcast::CommandBcast(const string &filename, char type) : Command((short)9), filename(filename), type(type){}

string CommandBcast::getFilename(){
	return filename;
}

char CommandBcast::getType(){
	return type;
}

CommandBcast::~CommandBcast(){

}
